lib/pyper/pipes/cassandra/reader.rb

```diff
@@ -21,12 +21,17 @@ def pipe(arguments, status = {})
       columns = arguments.delete(:columns)
       consistency = arguments.delete(:consistency)
 
+      # arguments to be searched by < and >
+      comparators = arguments.delete(:comparators)
```

Comment:
I'd propose changing this field to `:where` and allowing it to contain an arbitrary list of where clauses.

```rb
where = arguments.delete(:where)

query = client.select(table, columns)
query = query.where(arguments) if arguments.any? # backwards-compatibility -- it should be preferred to pass in `:where`
where.each { |clause| query = query.where(*clause) }
```
Example arguments would be like

```rb
arguments = { limit: 1, where: [ ['x > ? AND x < ?', 4, 6], ['x = ?' 5] ] }
```

That would be more general/flexible and be compatible with cassava currently.